### PR TITLE
[TAP 8] minor updates to the abstract

### PR DESCRIPTION
**Pull request by mnm678**
Some minor changes from my recent read-through

### CODE REVIEW

File: tap8.md [hunk 1]
```diff
@@ -21,17 +21,17 @@ by roles that have multiple trusted keys by allowing the role to define a new
...
 Performing a key rotation does not require parties that delegated trust to the
 role to change their delegation. Roles are thus able to rotate to new trusted
-keys independently, allowing keys to be changed more often.
+keys independently, allowing keys to be changed more often and without
+interaction from delegators.
 
-The key rotation mechanism is designed to be used by all roles except the
-root role. The root role will continue to use the root metadata to establish
-a trusted root key.
+The key rotation mechanism is designed to be used by any targets roles. The root role will continue to use the root metadata to establish a trusted root key.
```
Comment (JustinCappos) [inline tap8.md:27]:
```suggestion
The key rotation mechanism is designed to be used by any targets roles. The root 
role will continue to use the root metadata to establish a trusted root key.
```

**Review by JustinCappos — DISMISSED** [review]

File: tap8.md [hunk 2]
```diff
@@ -21,17 +21,18 @@ by roles that have multiple trusted keys by allowing the role to define a new
...
 role to change their delegation. Roles are thus able to rotate to new trusted
-keys independently, allowing keys to be changed more often.
+keys independently, allowing keys to be changed more often and without
+interaction from delegators.
 
-The key rotation mechanism is designed to be used by all roles except the
-root role. The root role will continue to use the root metadata to establish
-a trusted root key.
+The key rotation mechanism is designed to be used by any targets roles. The root 
+role will continue to use the root metadata to establish a trusted root key.
```
Comment (trishankatdatadog) [inline tap8.md:28]: What about `timestamp` and ` snapshot`?

Comment (mnm678) [reply tap8.md:28]: They aren't supported. We need the timestamp/snapshot to ensure that all rotate files are seen, and so can't use this mechanism for timestamp and snapshot.

Comment (trishankatdatadog) [reply tap8.md:28]: Maybe just add a sentence to clarify this here pls? Thx!

Comment (mnm678) [reply tap8.md:28]: done!

**Review by trishankatdatadog — APPROVED** [review]

File: tap8.md [hunk 3]
```diff
@@ -42,8 +44,7 @@ rotation to null without the help of the delegator, so they are able to
 explicitly revoke trust in the role immediately, improving response time
 to a key compromise. A rotation to a null key revokes trust in the role,
 not specific keys, so all keys associated with the role will be invalid
-after a rotation to null. If only a single key needs to be replaced, it can be
-safely rotated using the mechanism in this TAP. The client will detect a rotation
+after a rotation to null. The client will detect a rotation
```
Comment (JustinCappos) [inline tap8.md:47]: I thought that self revocation was moving to another TAP?

Comment (mnm678) [reply tap8.md:47]: Yes, that will be the next pr (which also adds the new TAP)

**Pull request merged**